docs/guides/cloudeventstarget.md

```diff
@@ -0,0 +1,57 @@
+# Exporting CloudEvents
+
+The Trigermesh `CloudEventsTarget` API object is used to export Cloudevents to external systems via HTTP.
+
+## Configuring a CloudEventsTarget Object
+
+The CloudEventsTarget accepts parameters to set the endpoint where CloudEvents will be sent and authentication. When succesfuly created it consumes and forwards to the configured an HTTP endpoint.
```

Comment:
"successfuly" => successfully

Reply:
Maybe can remove the "When succesfuly created it consumes and forwards to the configured an HTTP endpoint." sentence which is a bit awkward, maybe better place in the "usage" section?